### PR TITLE
scalability framework: regression handling

### PR DESCRIPTION
**Pull request by nrainer-materialize**
Content:
* 7752e5115e scalability framework: increase regression threshold
* 11dcb217a3 scalability framework: increase number of retries
* 0585c7cbe3 scalability framework: log retries

### CODE REVIEW

File: test/scalability/mzcompose.py [hunk 1]
```diff
@@ -42,6 +42,8 @@
     Postgres(),
 ]
 
+REGRESSION_THRESHOLD_IN_PERCENT = 0.2
```
Comment (philip-stoev) [inline test/scalability/mzcompose.py:45]: so 20? Or the name of the variable and the `max_deviation_in_percent` constructor argument are misleading?

**Review by philip-stoev — APPROVED** [review]

File: test/scalability/mzcompose.py (hunk 1 above)
Comment (def-) [reply test/scalability/mzcompose.py:45]: I vote for changing the variable name to something like `REGRESSION_THRESHOLD`

**Review by def- — APPROVED** [review]
Modulo Philip's comment

File: test/scalability/mzcompose.py (hunk 1 above)
Comment (nrainer-materialize) [reply test/scalability/mzcompose.py:45]: I am not entirely convinced (see also https://stackoverflow.com/questions/48585623/percentage-stored-as-a-fraction-of-1-what-is-this-called and https://stackoverflow.com/questions/4840095/naming-convention-for-0-0-1-0-value-range). `AS_PERCENT_DECIMAL`?

**Pull request merged**